src/test/java/com/example/rets_api/integration/RetsControllerIntegrationTest.java

```diff
@@ -124,10 +133,58 @@ public void deletetPropertyById() {
         assertEquals(404, responseEntityGetAgain.getStatusCodeValue());
     }
 
+    @Test
+    public void patchProperty() {
+        //Creating objects
+        PropertyDTO propertyDTORequest = createPropertyDTOWithBasicFields();
+        PropertyPatchDTO propertyPatchDTORequest = createPropertyPatchDTOWithBasicFields();
+        String URL = "http://localhost:" + port + "/properties";
+
+        //Request
+        ResponseEntity<Long> responseEntity = this.restTemplate
+                .postForEntity(URL, propertyDTORequest, Long.class);
+
+        //Creating objects
+        String URLWithID = URL
+                .concat("/")
+                .concat(String.valueOf(responseEntity.getBody()));
+
+        //Request
+        ResponseEntity<PropertyDTO> responsePatchEntity = this.restTemplate
+                .exchange(URLWithID,HttpMethod.PATCH, new HttpEntity<>(propertyPatchDTORequest), PropertyDTO.class);
+
+        //Validation
+        assertEquals(200, responsePatchEntity.getStatusCodeValue());
+        assertNotNull(responsePatchEntity.getBody());
+
+        //Validation
+        comparePropertyPatchDTOBasicFields(responsePatchEntity.getBody(), propertyPatchDTORequest);
+    }
+
+    @Test
+    public void patchPropertyNotExist() {
+        //Creating objects
+        PropertyPatchDTO propertyPatchDTORequest = createPropertyPatchDTOWithBasicFields();
+
+        String URL = "http://localhost:" + port + "/properties";
+
+        //Creating objects
+        String URLWithID = URL
+                .concat("/")
+                .concat(String.valueOf(1));
```

Comment:
please use some bigger value, because while the test is running the id 1 can be in memory... use like 123456